### ISSUE TITLE
Make type required

### ISSUE THREAD

**Issue by Saiv46** [siranweb/activitypub-types#2]
Types provide all properties but `@context` and `type`, that needs to be handled manually.

It would be nice if those properties would not be optional and instead TypeScript would help validate the schema.

**Comment by siranweb** [comment]
Yeah, you're right about `type`. But with `@context` It's not that simple.

There is an [example](https://www.w3.org/TR/activitystreams-core/#example-2) in docs that some objects can have nested objects with no `@context` field. The [same think](https://github.com/siranweb/activitypub-types/blob/main/src/model-interfaces/activities/activity.interface.ts#L19) is made in types too.

As a workaround you can do something like that:
```ts
type APRoot<APModel> = APModel & {
    '@context': string;
}

const activity: APRoot<APActivity> = {
    '@context': 'https://www.w3.org/ns/activitystreams', // now this required
    // ...
}
```

I'll make `type` field required. For `@context` probably I'll add such `APRoot` type and I'll remove it from `APObject`

**Comment by Saiv46** [comment]
I'm already implementing `type` field. But yeah, `@context` is optional in nested types.

**Comment by siranweb** [comment]
Will be released in v1.1.0
